scripts/future-auto-shutdown.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
```

Comment:
what's this script?

Reply:
Its to replace the existing shutdown script when we are ready for it. Mohana working a couple of tickets that need to be finished first

Reply:
i think this should be integrated in the main script, what's remaining?

This script is looking awfully long and may need refactoring or possibly writing in another language.

Reply:
Agree we should integrate with the existing shutdown script. 

In terms of length, its shorter than the auto-start script and will get a bit shorter during the integration with the main script. Given the other scripts are in bash I would be keen to keep the shutdown in bash as well.